FrontEnd/Profile SignUp Page/src/SignUp/SignUpPageContent/SignUpForm/SignUpFormContent.js

```diff
@@ -0,0 +1,121 @@
+import css from './SignUpFormContent.module.css'
+import { useState } from 'react';
+import EmailFieldComponent from './Fields/EmailField';
+import PasswordFieldsComponent from './Fields/PasswordFields';
+import FullNameFieldsComponent from './Fields/FullNameFields';
+import CompanyFieldsComponent from './Fields/CompanyFields';
+import BottomCheckboxesComponent from './Fields/BottomCheckboxes';
+
+
+function SignUpFormContentComponent() {
+
+    const [formContent, setFormContent] = useState({
+        'email': '',
+        'password': '',
+        'confirmPassword': '',
+        'surname': '',
+        'name': '',
+        'companyName': '',
+        'companyCheckbox': false,
+        'startupCheckbox': false,
+        'rulesAgreement': false
+    });
+
+    const [formState, setFormState] = useState({
+        email: {
+            'error': false,
+            'message': ''
+        },
+        password: {
+            'error': false,
+            'message': ''
+        },
+        confirmPassword: {
+            'error': false,
+            'message': ''
+        },
+        surname: {
+            'error': false,
+            'message': ''
+        },
+        name: {
+            'error': false,
+            'message': ''
+        },
+        companyName: {
+            'error': false,
+            'message': ''
+        },
+    });
+
+    const checkRequiredFields = (formData) => {
+        const newFormState = {};
+        for (const key in formData) {
+            if (!formData[key]) {
+                newFormState[key] = {
+                    'error': true,
+                    'message': 'Обов’язкове поле',
+                }
+            } else {
+                newFormState[key] = {
+                    'error': false,
+                    'message': '',
+                }
+            }
+        }
+        const nextFormState = {
+            ...formState,
+            ...newFormState,
+        }
+
+        setFormState(nextFormState);
```

Comment:
Where does this data then go? I do not understand

Reply:
formState then passed to the field components as error data , error message to be precise.  email -> lines 97, passwords ->  102-102, fullname -> 108-109, company -> 113

Reply:
Ok, I just thought that there is going to be some data storage for this project, to store user login info;
marking this as resolved then.